.gitignore

```diff
@@ -2,9 +2,11 @@ bin/
 *.db
 *.elf
 *.core
+.DS_Store
```

Comment:
I've been chastised for adding these in the past with the suggestion to add them to a global .gitignore on my own machine.